### PR TITLE
Fix creating a demo with no patches

### PR DESCRIPTION
**Pull request by MatmaRex**

### CODE REVIEW

File: new.php [hunk 1]
```diff
@@ -116,12 +119,24 @@
 	}
 }
 
-$patchesAppliedText = implode( ' ', $patchesApplied );
+$branchDesc = preg_replace( '/^origin\//', '', $branch );
+
+$wikiName = "Patch Demo (" . trim(
+	// Add branch name if it's not master, or if there are no patches
+	( $branchDesc !== 'master' || !$patchesApplied ? $branchDesc : '' ) . ' ' .
```
Comment (edg2s) [inline new.php:126]: Put the extra space inside the ternary, otherwise you will get  "( 123456)", and "(REL1.35 )"

**Comment by edg2s** [comment]
I suppose you can now create master+nopatches (for reasons unknown) which will have no title and look a little odd in the table.

File: new.php (hunk 1 above)
Comment (MatmaRex) [reply new.php:126]: There's the `trim()` to fix that. (It's annoying to do this without it, because either "side" can be empty.)

**Comment by MatmaRex** [comment]
It will have just "(master)" in the title, but you're right I forgot about the table on the main page.

**Comment by MatmaRex** [comment]
I think that could still be useful for comparing the behavior before/after some patch, so I didn't disallow that.

**Pull request merged**